.gitignore

```diff
@@ -46,6 +46,7 @@ manifest.json
 /pootle/locale/*/LC_MESSAGES/
 /pootle/log/
 /pootle/static/translations/
+/pootle/invoices/
```

Comment:
This seems to be related to how you set things up, not to the project code itself. Let's remove this rule from the PR; you can always add personal ignore rules to .git/info/exclude file in your local checkout.

Reply:
Didn't knew there was a local exclude file. Great then

Reply:
👌